### PR TITLE
fix(QB-2081): Remove unsupported number formats

### PR DESCRIPTION
**Pull request by johanlahti**
I have investigated and found that only auto and number formats are working in the Radar chart, so **removing the other options in this PR**. If we implement this later on (which seems to require a bit more code than expected) we should probably use Picasso's number formatting functionality, which the Picasso charts rely on today – instead of maintaining a parallel implementation for this chart only.

**Other code changes in this PR:**
- prevent custom format for number option
- lint fixes
- removed one unneeded else after return
- used webpack mode instead of referring to it again

### CODE REVIEW

**Review by Caele — APPROVED** [review]
Yea, sounds resonable.
Did you check/understand why it doesn't use the formatting from engine?

**Comment by johanlahti** [comment]
I did not check that, I assume it would still need the client-side formatting though?
I will create a Jira task for adding formatting. Maybe you can add a comment about this and optional implementations (e.g. using Picasso).

**Comment by johanlahti** [comment]
Btw I don't have write access so I cannot merge this PR 😄

**Pull request merged**
